test/plugin/test_in_forward.rb

```diff
@@ -7,6 +7,22 @@
 require 'fluent/plugin/in_forward'
 
 class ForwardInputTest < Test::Unit::TestCase
+  class << self
+    def startup
+      if Fluent.windows?
+        socket_manager_path = ServerEngine::SocketManager::Server.generate_path
+      else
+        socket_manager_path = ServerEngine::SocketManager::Server.generate_path + __method__.to_s
+      end
```

Comment:
Could you reply here about what the difference between windows and else?
In other words, I want to know the difference between `Server.generate_path` and `Server.generate_path + __method__.to_s` and why (to write plugin helper for these sockets).